Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
Automatic builds are compatible with having this `docker` directory in the sense that you can point to the concrete location of the `Dockerfile` including a path from the root of the repo.
I was mostly thinking on the case when a developer clones the Github repo and uses the `Dockerfile` to build the image. It has 2 options: 1) Building it from the root of the repo or 2) doing it from inside the `docker` folder (of course she could do it from any other path in her HD :) ) but I think a `docker` folder would invite her to do it from there :) The point is that the paths used in the `Dockerfile` depends on the path from where she runs the `docker build` command, right?
That's the reason why maybe including this `docker` folder may create more problems than solve them :)